sa3ed_app/sa3ed_app/doctype/lost_person_task/templates/lost_person_task.html

```diff
@@ -0,0 +1,25 @@
+{% extends "templates/web.html" %}
+
+{% block page_content %}
+<h1>{{ title }}</h1>
+<div class="row align-items-center justify-content-center" style="margin-bottom: 30px; margin-top: 30px; gap: 30px;">
+    <div class="col">
+        <img src="{{image}}" alt="" style="border-radius: 30px;">
+    </div>
+
+    <div class="col text-dark">
+        <div class="row badge badge-danger" style="width: min-content; padding: 6px 10px; margin-bottom: 7px; transform: translateX(-40px);">
+            Status: {{status}}
+        </div>
+        <div class="row" style="margin-bottom: 7px;">Name: {{name1}}</div>
```

Comment:
display the label as First Name instead of Name. and you should use multilang keys to add support for multi-languages, example: Name change to {{ _("Name") }}